### PR TITLE
[CBRD-25172] support %TYPE in SP parameter and return types

### PR DESCRIPTION
**Pull request by hyunikn**
http://jira.cubrid.org/browse/CBRD-25172

### CODE REVIEW

File: msg/en_US.utf8/cubrid.msg [hunk 1]
```diff
@@ -1415,8 +1415,8 @@ Check the path of the key file (_keys) and make sure that it includes proper key
 1337 Log record does not exist (lsa : %1$lld|%2$d). The archive log volume has been deleted.
 1338 Flashback is already running. Flashback cannot be run by more than one user at the same time.
 
-1339 Unsupported argument type of stored procedure: %1$s
-1340 Unsupported return type of stored procedure: %1$s
+1339 (deleted)
```
Comment (ctshim) [inline msg/en_US.utf8/cubrid.msg]: Trivial;
  Although the intention is slightly different, currently unused but reserved numbers are expressed in the following way.
"`Error reserved for...`"

File: src/parser/csql_lexer.l [hunk 2]
```diff
@@ -1015,9 +1015,12 @@ IDL       [a-zA-Z0-9_]
 										csql_yylval.cptr = pt_makename(yytext);
 										return THAN; }
 [tT][iI][mM][eE]							{ begin_token(yytext);   return Time; }
-[tT][iI][nN][yY][iI][nN][tT]				{ begin_token(yytext);   return SmallInt; }
+[tT][iI][nN][yY][iI][nN][tT]				                { begin_token(yytext);   return SmallInt; }
```
Comment (ctshim) [inline src/parser/csql_lexer.l:1018]: You need to register "tinyint" as a reserved word.

File: msg/en_US.utf8/cubrid.msg (hunk 1 above)
Comment (hyunikn) [reply msg/en_US.utf8/cubrid.msg]: The intention seems slightly different. Mine was actually for  error numbers which are no more used and reusable by anyone,

File: src/parser/csql_lexer.l (hunk 2 above)
Comment (hyunikn) [reply src/parser/csql_lexer.l:1018]: No changes were made to the existing TINYINT keyword except for the indentation. 
The new keyword, TYPE, introduced in this PR was added to the derivations from Identifier and to the keyword list in keyword.c. 
Thank you.

**Review by ctshim — APPROVED** [review]

**Review by hgryoo — APPROVED** [review]

**Review by beyondykk9 — APPROVED** [review]

**Pull request merged**